pkg/az/action_test.go

```diff
@@ -38,5 +38,5 @@ func TestStep_GetFlags(t *testing.T) {
 	f := s.GetFlags()
 
 	require.Len(t, f, 1, "Flags should always have at least 1 entry: --out")
```

Comment:
Can you update the the test error message to match that we now insert `--output` by default instead of `--out`?

Reply:
Done!